src/include/OpenImageIO/errorhandler.h

```diff
@@ -83,16 +83,37 @@ class OIIO_API ErrorHandler {
     }
     virtual ~ErrorHandler() {}
 
+    /// Set desired verbosity level.
+    void verbosity(int v) { m_verbosity = v; }
+
+    /// Return the current verbosity level.
+    int verbosity() const { return m_verbosity; }
+
     /// The main (or "full detail") method -- takes a code (with high
     /// bits being an ErrCode) and writes the message, with a prefix
     /// indicating the error category (no prefix for "MESSAGE") and
     /// error string.
     virtual void operator()(int errcode, const std::string& msg);
 
-    /// Info message with printf-like formatted error message.
-    /// Will not print unless verbosity >= VERBOSE.
+    // Base cases -- take a single string
```

Comment:
Are these base cases necessary?

Reply:
Or at the very least -- can we use `string_view` here?

Reply:
If we use string_view, and the caller passes a string, it will have to make a second string when it passes along to `operator()`. In an ideal world, we would have made `operator()` take a string_view, but we established ErrorHandler's interface before we had string_view. And every renderer that uses OSL supplies or uses ErrorHandler, so we can't change the interface now.  But remember, this is an error handler, it's only called rarely, it's ok if there are a few extra string copies.

The base cases are unfortunately necessary because people (including, um, me, in OSL, sorry) sometimes pass just one string to send an error that's not supposed to be formatted at all. If that should contain a "%" or someday a "{}", it will go kablooey if there isn't the right kind of argument following.